packages/shared/src/components/squads/Comment.tsx

```diff
@@ -70,21 +82,22 @@ export function SquadComment({
     {
       onSuccess: (postByUrl) => {
         if (postByUrl) {
-          onUpdateForm(postByUrl);
+          onUpdateForm({ post: { post: postByUrl } });
           textinput?.current?.focus();
         }
       },
-      onError: (err: ApiErrorResult) => {
+      onError: (err: ApiErrorResult, url) => {
         if (
           err?.response?.errors?.[0].extensions.code === ApiError.NotFound &&
           !isNullOrUndefined(postItem)
         ) {
-          onUpdateForm(null);
+          onUpdateForm({ post: { post: null } });
```

Comment:
Should we not show an error if user hits rate-limit or the url is not an actual URL or something like that?

Reply:
Yes, that is not yet implemented as we're still waiting for the API PR.

Reply:
Since we know what to expect, whenever it fails we just remove the preview.